kedro-telemetry/RELEASE.md

```diff
@@ -6,21 +6,21 @@
 
 The `kedro-telemetry` plugin asks for consent to collect command run data. However, when we introduced functionality to understand the size of projects by counting the number of datasets, pipelines and nodes, the plugin did not verify consent, leading to the plugin collecting data without user consent. This bug affects users with `kedro-telemetry >= 0.2.2` installed, 22.5% of our user base.
 
-<img width="730" alt="Screenshot 2023-04-04 at 16 17 43" src="https://user-images.githubusercontent.com/43755008/229840293-fea91403-b02d-4221-9167-71d3a6196cc5.png">
+<img width="730" src="https://user-images.githubusercontent.com/43755008/229840293-fea91403-b02d-4221-9167-71d3a6196cc5.png">
 
 Here is an example of what was collected.
 
-|Description|What we received|
-|-|-|
-|_(Hashed)_ Package name| c7cd944c28cd888904f3efc2345198507...|
-|_(Hashed)_ Project name| a6392d359362dc9827cf8688c9d634520e...|
-|`kedro` project version| 0.18.4|
-|`kedro-telemetry` version| 0.2.3|
-|Python version| 3.8.10 (default, Jun  2 2021, 10:49:15) |
-|Operating system used| darwin |
-| Number of datasets | 7 |
-| Number of pipelines | 2 |
-| Number of nodes | 12 |
+| Description               | What we received                        |
```

Comment:
I see no difference in markup before/after

Reply:
The rendering is the same. The spacing makes the table readable in text mode in the editor.